diff -Nru usbmuxd-1.0.9/configure.ac usbmuxd-1.0.9.orig/configure.ac
--- usbmuxd-1.0.9/configure.ac	2014-03-29 22:14:20.000000000 +0000
+++ usbmuxd-1.0.9.orig/configure.ac	2014-04-08 09:44:33.000000000 +0100
@@ -84,6 +84,7 @@
 Makefile
 src/Makefile
 udev/39-usbmuxd.rules
+udev/usbmuxd.service
 ])
 
 echo "
diff -Nru usbmuxd-1.0.9/Makefile.am usbmuxd-1.0.9.orig/Makefile.am
--- usbmuxd-1.0.9/Makefile.am	2014-03-29 22:14:20.000000000 +0000
+++ usbmuxd-1.0.9.orig/Makefile.am	2014-04-08 10:09:08.000000000 +0100
@@ -4,3 +4,5 @@
 
 udevrulesdir = /lib/udev/rules.d
 udevrules_DATA = udev/39-usbmuxd.rules
+systemdunitdir = /lib/systemd/system
+systemdunit_DATA = udev/usbmuxd.service
diff -Nru usbmuxd-1.0.9/udev/39-usbmuxd.rules.in usbmuxd-1.0.9.orig/udev/39-usbmuxd.rules.in
--- usbmuxd-1.0.9/udev/39-usbmuxd.rules.in	2014-03-29 22:14:20.000000000 +0000
+++ usbmuxd-1.0.9.orig/udev/39-usbmuxd.rules.in	2014-04-08 08:46:57.000000000 +0100
@@ -1,7 +1,7 @@
 # usbmuxd ("Apple Mobile Device" muxer listening on /var/run/usbmuxd)
 
 # Initialize iOS devices into "deactivated" USB configuration state and run usbmuxd
-ACTION=="add", SUBSYSTEM=="usb", ATTR{idVendor}=="05ac", ATTR{idProduct}=="12[9a][0-9a-f]", ENV{USBMUX_SUPPORTED}="1", ATTR{bConfigurationValue}="0", OWNER="usbmux", RUN+="/@prefix@/sbin/usbmuxd -u -U usbmux"
+ACTION=="add", SUBSYSTEM=="usb", ATTR{idVendor}=="05ac", ATTR{idProduct}=="12[9a][0-9a-f]", ENV{USBMUX_SUPPORTED}="1", ATTR{bConfigurationValue}="0", OWNER="usbmux", TAG+="systemd", ENV{SYSTEMD_WANTS}="usbmuxd.service"
 
 # Exit usbmuxd when the last device is removed
-ACTION=="remove", SUBSYSTEM=="usb", ENV{PRODUCT}=="5ac/12[9a][0-9a-f]/*", ENV{INTERFACE}=="255/*", RUN+="@prefix@/sbin/usbmuxd -x"
+ACTION=="remove", SUBSYSTEM=="usb", ENV{PRODUCT}=="5ac/12[9a][0-9a-f]/*", ENV{INTERFACE}=="255/*", RUN+="/usr/sbin/usbmuxd -x"
diff -Nru usbmuxd-1.0.9/udev/usbmuxd.service.in usbmuxd-1.0.9.orig/udev/usbmuxd.service.in
--- usbmuxd-1.0.9/udev/usbmuxd.service.in	1970-01-01 01:00:00.000000000 +0100
+++ usbmuxd-1.0.9.orig/udev/usbmuxd.service.in	2014-04-08 09:57:09.000000000 +0100
@@ -0,0 +1,7 @@
+[Unit]
+Description=iOS USB Muxer
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/usbmuxd -u -U usbmux
+PIDFile=/var/run/usbmuxd.pid
