--- usbmuxd-1.0.9.orig/src/client.c	2014-04-14 01:00:19.000000000 +0100
+++ usbmuxd-1.0.9/src/client.c	2014-04-14 10:31:11.446913875 +0100
@@ -476,6 +476,11 @@
 			} else {
 				char *message = NULL;
 				plist_t node = plist_dict_get_item(dict, "MessageType");
+				if (!node) {
+					usbmuxd_log(LL_ERROR, "plist_dict_get_item failed!");
+					plist_free(dict);
+					return -1;
+				}		
 				plist_get_string_val(node, &message);
 				if (!message) {
 					usbmuxd_log(LL_ERROR, "Could not extract MessageType from plist!");
